Add new topic to describe how to allow multiple admin login (#1357)
magento/merchdocs: src/stores/admin-signin.md

The Sign In page displays a message that you are logged out. We recommend you sign out of the _Admin_ any time you leave your computer unattended.

> ## Allow Multiple Admin Login

Reviewer: ```suggestion
## Allow multiple Admin logins
```

> The admin manages the orders, customers, products, shipping, and payments functionality. The default configuration is set to not allowing the admin account sharing for security purposes. Multiple admin users can manage the store faster and distribute the workload for better efficiency.

Reviewer: This function actually determines if an account can be logged in at the same time from different devices. 

```suggestion
The Admin provides access to manage the orders, customers, products, shipping, and payments functionality. The default configuration is set to disallow multiple logins for an Admin user account as a security best practice. However, you can change this setting to allow Admin users to be logged in from multiple devices if that is needed to accommodate your business workflows.
```

> 1. On the _Admin_ sidebar, go to **Stores** > **Configuration**.

Reviewer: ```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
```

    ![]({% link images/images/multiple-admin-login.png %}){: .zoom}
    _Multiple Admin Login_

> 1. Under **Advanced**, select **Admin**.

Reviewer: ```suggestion
1. In the left panel, expand **Advanced** and choose **Admin**.
```

> 1. Expand the **Security** section.

Reviewer: ```suggestion
1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Security** section.
```

> 1. Set `Yes` in the **Admin Account Sharing** field.

Reviewer: ```suggestion
1. For **Admin Account Sharing**, select `Yes`.
```

> 1. Click **Save** the configuration.

Reviewer: ```suggestion
1. Click <span class="btn">Save Config</span>.
```

Author: Done, thank you🙏😊
